Server/App/model/card.py

```diff
@@ -16,9 +16,23 @@ def create(deck_id, front, back, position, reposition=True, connector=None):
         connector.call_procedure_transactionally(procedure_name, uuid, deck_id, front, back, position)
         return Card(uuid, connector)
 
-    def edit(self, deck_id, front, back, position):
-        self.connector.call_procedure_transactionally(
-            'EDIT_CARD', self.uuid, deck_id, front, back, position)
+    def edit(self, deck_id, front=None, back=None):
+        query = 'UPDATE Card SET {} WHERE deck_id=%s AND uuid=%s'
+        sub_list = []
+        arg_list = []
+        if front is not None:
+            sub_list.append('front=%s')
+            arg_list.append(front)
+        if back is not None:
+            sub_list.append('back=%s')
+            arg_list.append(back)
+
+        if len(sub_list) > 0:
+            self.connector.query_transactionally(query.format(','.join(sub_list)),
+                                                 *(arg_list + [deck_id, self.uuid]))
```

Comment:
Although now that I think about it, you could just select the card first, and then update the card to be the new value, provided param is not null, or set it back to the old value if its null.